platform_channels/ios/Runner/AccelerometerStreamHandler.swift

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import CoreMotion
+
+class AccelerometerStreamHandelr: NSObject, FlutterStreamHandler {
```

Comment:
nit: Class name is misspelled (should end with "Handler").

Reply:
Done :D